rmf_site_format/src/workcell.rs

```diff
@@ -289,6 +344,110 @@ impl Workcell {
         serde_json::ser::to_string_pretty(self)
     }
 
+    pub fn to_urdf(&self) -> Result<urdf_rs::Robot, WorkcellToUrdfError> {
+        let workcell = self.clone();
```

Comment:
I'm a bit on the fence on the early `clone` here, while it makes the code below a lot simpler it could create unnecessary copies of data, compared to only iterating through references in `self` and ad-hoc cloning of only necessary variables. This might not matter too much now but if in the future we added new fields to workcells that don't map to URDF, this `clone()` call would copy all that data unnecessarily as well. 

Also, visuals and collisions are actually cloned twice, once in this line and once [here](https://github.com/audrow/rmf_site/blob/5e376ea789f44311b5e3bfdc10d086c7cd8eb067/rmf_site_format/src/workcell.rs#L392). This sounds unnecessary for what is a single export.

I can be convinced that the early clone to save on all the `.clone()` calls scattered throughout the function can be worth it, but we should at least get rid of the double clone (which should be addressed if you go for the hash map approach).

Reply:
I tried to remove the unnecessary clone calls in 41e04ff.